src/components/donation/Steps/Submit/StripeCheckout/StripeCheckout.tsx

```diff
@@ -64,8 +64,22 @@ export default function StripeCheckout(props: StripeCheckoutStep) {
         ) : (
           <Elements
             options={{
+              fonts: [
+                {
+                  family: "DM Sans",
+                  cssSrc: "https://fonts.googleapis.com/css2?family=Quicksand",
+                },
+              ],
               clientSecret,
-              appearance: { theme: "stripe" },
+              appearance: {
```

Comment:
styling tweaks for stripe
<img width="648" alt="image" src="https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/89639563/26ce8def-efe0-47fc-805b-81e8877b4058">